src/probe_matcher.cpp

```diff
@@ -126,7 +126,7 @@ std::set<std::string> ProbeMatcher::get_matches_for_probetype(
       break;
     }
     case ProbeType::tracepoint:
-    {
+    case ProbeType::rawtracepoint: {
```

Comment:
I don't think that this is correct. `/sys/kernel/tracing/available_events` contains entries in the form `category:event` while raw tracepoints only use the `event` part.

Listing all raw tracepoints shows an empty list:
```
# bpftrace -l 'rawtracepoint:*`
#
```
As I said above, I suggest removing wildcard support for now and adding it in a follow-up PR.